src/components/NavBar/NavBar.jsx

```diff
@@ -143,10 +143,10 @@ const NavBar = () => {
                 />
               )}
               {/* modal/popup renders when showConfirmationModal state is true */}
-              <LogoutModal
```

Comment:
perfect example of the point...

If the name is now just `Modal` and the function is now just `handle`   -  "what IS this modal and what does it do?"  and you'd have to dig around to find out.

Reply:
and as another point, you didn't do this in `ClientListTableRow`. There you have the component named `<DeleteModal>`, codebase should be consistent one way or the other - but it's ideal to be specific.